pkgs/development/tools/ilspycmd/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+, powershell
+, autoSignDarwinBinariesHook
+, glibcLocales
+}:
+buildDotnetModule rec {
+  pname = "ilspycmd";
+  version = "8.0";
+
+  src = fetchFromGitHub {
+    owner = "icsharpcode";
+    repo = "ILSpy";
+    rev = "v${version}";
+    sha256 = "ERBYXgpBRXISfqBSBEydEQuD/5T1dvJ+wNg2U5pKip4=";
```

Comment:
Would you please use SRI hash?
> Nixpkgs contributors are currently recommended to use hash.

https://nixos.org/manual/nixpkgs/unstable/#fetchurl